src/index.js

```diff
@@ -0,0 +1,193 @@
+
+const actualTempValue = document.getElementById('actualValue')
+
+const increaseTempControl = document.getElementById('increaseTempControl');
+
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+
+const skySelect = document.getElementById('skySelect');
+
+const desiredSky = document.getElementById('desiredSky')
+
+const desiredLandscape = document.getElementById('landscape')
+
+const bgColor = document.getElementById('backgroundColor')
+
+const headerCityName = document.getElementById('headerCityName')
+
+const cityNameInput = document.getElementById('cityNameInput')
+
+const submitButton = document.getElementById('cityNameSubmit')
+
+const resetButton = document.getElementById('cityNameReset')
+
+
+
+// Functions 
+// TEMP CHANGES
+const increaseTemp = () => {
+    state.temp ++;
+    tempValue.innerHTML = state.temp + "°" ;
+} 
+
+const decreaseTemp = () => {
+    state.temp --;
+    tempValue.innerHTML = state.temp + "°" ;
+}
+
+// CITY HEADER
+const changeCityName = () => {
+    headerCityName.innerHTML = cityNameInput.value 
+
+}
+
+// State.Location Update
+const locationUpdate = () => {
+    state.location = cityNameInput.value; 
+    getLocation();
+}
+
+const resetLocation = () => {
+    state.location = '';
+    cityNameInput.value = state.location;
+    changeCityName();
+}
+
+//SKY CHANGES 
+const skySelector = () => {
+    if (state.sky.value == "sunny") {
+        desiredSky.textContent = "☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞☀️🌞"
+    }
+    else if (state.sky.value == "cloudy") {
+        desiredSky.textContent = "🌥🌥🌥🌥🌥🌥🌥🌥🌥🌥🌥"
+    }
+    else if (state.sky.value == "rainy") {
+        desiredSky.textContent = "🌧🌧🌧🌧🌧🌧🌧🌧🌧🌧🌧"
+    }
+    else if (state.sky.value == "snowy") {
+        desiredSky.textContent = "🌨❄️🌨❄️🌨❄️🌨❄️🌨❄️🌨❄️🌨❄️🌨"
+    }
+}
+
+const validateSkyCondition = (skyCondition) => {
+    if (skyCondition == "Thunderstorm" || skyCondition == "Drizzle" || skyCondition == "Rain" ) {
+        state.sky.value = "rainy";
+    } 
+    else if (skyCondition == "Clouds") {
+        state.sky.value = "cloudy";
+    }
+    else if (skyCondition == "Snow") {
+        state.sky.value = "snowy";
+    }
+    else {
+        state.sky.value = "sunny";
+    }
+}
+
+// LANDSCAPE CHANGES
+const landscapeChange = () =>{
+    if (state.temp >= 80){
+        desiredLandscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+        bgColor.style.backgroundColor = '#E24E1B'
+    } 
+    else if (state.temp >= 70 ){
+        desiredLandscape.textContent = "🌸🌿🌼__🌷🌻🌿_🌱_🌻🌷"
+        bgColor.style.backgroundColor = '#E5B25D'
+    }
+    else if (state.temp >= 60 ){
+        desiredLandscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+        bgColor.style.backgroundColor = '#D0CFEC'
+    }
+    else if (state.temp >= 50 ){
+        desiredLandscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+        bgColor.style.backgroundColor = "#875C74"
+    }
+    else if (state.temp <= 40 ){
+        bgColor.style.backgroundColor = "#1B4079"
+    }
+}
```

Comment:
There’s an issue with wave 02 (temperature), the requirements specify that the temperature ranges should change the text color or the background of the temperature & the landscape depending on what temperature it is. Currently your code is updating the background color if the entire page.